libs/SalesforceHybridSDK/SalesforceHybridSDK/Classes/Plugins/SFNetworkPlugin/SFNetworkPlugin.m

```diff
@@ -92,7 +91,7 @@ - (void) pgSendRequest:(CDVInvokedUrlCommand *) command
             NSString* fileUrl = [fileParam nonNullObjectForKey:kFileUrl];
             NSString* fileName = [fileParam nonNullObjectForKey:kFileName];
             NSData* fileData = [NSData dataWithContentsOfURL:[NSURL URLWithString:fileUrl]];
-            [request addPostFileData:fileData paramName:fileParamName fileName:fileName mimeType:fileMimeType];
+            [request addPostFileData:fileData description:nil fileName:fileName mimeType:fileMimeType];
```

Comment:
Setting `description` doesn't seem to be respected or supported in `v39.0` for regular file upload, based on the generic file upload example [here](https://developer.salesforce.com/docs/atlas.en-us.chatterapi.meta/chatterapi/intro_input.htm), in our official API doc.